frontend/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
```

Comment:
Might want to consider having a linter configuration in the parent directory, which could be the base for all others.